CHANGELOG_PENDING.md

```diff
@@ -1,45 +1,67 @@
-- [abci] \#2159 Update use of `Validator` ala ADR-018:
-    - Remove PubKey from `Validator` and introduce `ValidatorUpdate`
-    - InitChain and EndBlock use ValidatorUpdate
-    - Update field names and types in BeginBlock
-- [state] Implement BFT time
-- [p2p] update secret connection to use a little endian encoded nonce
-- [libs/clist] Panics if list extends beyond MaxLength
-- [common] SplitAndTrim was deleted
+
+* CLI/RPC/Config
+
+    - [config] \#2169 Replace MaxNumPeers with MaxNumInboundPeers and MaxNumOutboundPeers
+    - [config] \#2300 Reduce default mempool size from 100k to 5k, until ABCI rechecking is implemented.
+    - [rpc] \#1815 `/commit` returns a `signed_header` field instead of everything being
+      top-level
+
+* Apps
+
+    - [abci] Added address of the original proposer of the block to Header
+    - [abci] Change ABCI Header to match Tendermint exactly
+    - [abci] \#2159 Update use of `Validator` (see ADR-018):
+        - Remove PubKey from `Validator` (so it's just Address and Power)
+        - Introduce `ValidatorUpdate` (with just PubKey and Power)
+        - InitChain and EndBlock use ValidatorUpdate
+        - Update field names and types in BeginBlock
+    - [state] \#1815 Validator set changes are now delayed by one block
+        - updates returned in ResponseEndBlock for block H will be included in
+        RequestBeginBlock for block H+2
+
+* Go API
+
+    - [lite] \#1815 Complete refactor of the package
+    - [node] \#2212 NewNode now accepts a `*p2p.NodeKey`
+    - [libs/common] \#2199 Remove Fmt, in favor of fmt.Sprintf
+    - [libs/common] SplitAndTrim was deleted
+    - [libs/clist] Panics if list extends beyond MaxLength
+
+* Protocol
+
+    - [types] Updates to the Header:
+        - \#1815 NextValidatorsHash
+        - \#2106 ProposerAddress
+    - [blockchain] \#2213 Fix Amino routes for blockchain reactor messages
+    - [state] \#1815 Validator set changes are delayed by one block (!)
+        - Add NextValidatorSet to State, changes on-disk representation of state
+    - [consensus] \#2203 Implement BFT time
+        - Timestamp in block must equal median of timestamps in block's LastCommit
+    - [p2p] \#2263 Update secret connection to use a little endian encoded nonce
+    - [crypto] \#2205 Rename AminoRoute variables to no longer be prefixed by signature type.
+    - [crypto] \#2239 Secp256k1 signature changes (See ADR-014):
+        - format changed from DER to `r || s`, both little endian encoded as 32 bytes.
+        - malleability removed by requiring `s` to be in canonical form.
+
 
 FEATURES:
-- [types] allow genesis file to have 0 validators ([#2015](https://github.com/tendermint/tendermint/issues/2015))
-- [libs] allow passing options through when creating instances of leveldb dbs ([#2292](https://github.com/tendermint/tendermint/issues/2292))
+- [types] Allow genesis file to have 0 validators ([#2015](https://github.com/tendermint/tendermint/issues/2015))
```

Comment:
Should we make #2015 follow the same format as everything else? (i.e. be at the front)